src/tools/compiletest/src/runtest.rs

```diff
@@ -1432,6 +1433,12 @@ impl<'test> TestCx<'test> {
 
         aux_rustc.arg("-L").arg(&aux_dir);
 
+        if aux_props.add_core_stubs {
+            let minicore_path = self.build_minicore();
+            aux_rustc.arg("--extern");
+            aux_rustc.arg(&format!("minicore={}", minicore_path));
+        }
```

Comment:
What's going on here?

Reply:
Without this code added, the `//@ add-core-stubs` attribute silently does nothing in an auxillary crate.